### ISSUE TITLE
Slugify directories when doing handle_nav_based_import

### ISSUE THREAD

**Issue by nejch** [jdoiro3/mkdocs-multirepo-plugin#24]
Hi @jdoiro3 thanks for the plugin!

When using nav-based imports, the section naming defines the directory location and that is reflected in the final URL, unless I've missed a config option. It'd be nice if we could slugify those to have consistent URLs.

Would you accept that as an additional config option e.g. `slugify_dirs`? If so, where would you like that logic handled - by overriding `location` when initializing `DocsRepo` or is there a better place for this? :)

Basically this would be the opposite of what mkdocs does with titles when no `nav:` is present (it capitalizes them automatically).
